Pipfile

```
@@ -4,14 +4,15 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-pydle = {editable = true, ref = "asyncio", git = "git://github.com/Shizmob/pydle.git"}
+pydle = {editable = true,ref = "asyncio",git = "git://github.com/Shizmob/pydle.git"}
```

we can adjust this now that Pydle has a feature update.

```suggestion
pydle = ">=0.8.5"
```